util/src/main/java/tc/oc/pgm/util/material/Materials.java

```diff
@@ -52,6 +52,17 @@ public interface Materials {
       .addNullable(Material.getMaterial("MACE"))
       .build();
 
+  MaterialMatcher DOOR_ITEMS = MaterialMatcher.builder()
+      .addAll(m -> m.name().contains("DOOR") && !m.name().contains("TRAPDOOR") && !m.isBlock())
```

Comment:
```suggestion
      .addAll(m -> m.name().contains("_DOOR") && !m.isBlock())
```
wouldn't this do the same and avoid the two contains checks?